bitnami/vault/templates/injector/deployment.yaml

```
@@ -105,7 +105,7 @@ spec:
             - name: AGENT_INJECT_LISTEN
               value: :{{ .Values.injector.containerPorts.https }}
             - name: AGENT_INJECT_VAULT_ADDR
-              value: http://{{ include "vault.server.fullname" . }}:{{ .Values.server.service.general.ports.http }}
+              value: http://{{ include "vault.server.fullname" . }}.{{ include "common.names.namespace" . }}.svc:{{ .Values.server.service.general.ports.http }}
```

Why don't use the FQDN?

```suggestion
              value: http://{{ include "vault.server.fullname" . }}.{{ include "common.names.namespace" . }}.svc.{{ .Values.clusterDomain }}:{{ .Values.server.service.general.ports.http }}
```

---

I went with what they use in the [official chart](https://github.com/hashicorp/vault-helm/blob/757ae2d712f4f88acf379af2f686a3c4753f4f33/templates/injector-deployment.yaml#L67).
Also, this way it's consistent with `AGENT_INJECT_TLS_AUTO_HOSTS` and it works with the default `values.yaml` even if the cluster domain is not `cluster.local`.